test/e2e-common.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script includes common functions for testing setup and teardown.
+
+source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
+
```

Comment:
```suggestion

function knative_setup() {
  start_latest_knative_serving
}

```

Reply:
Not sure if this is what's desired, but just in case

Reply:
I'm adding that, but it's slightly smaller than what Serving has.